test/Microsoft.Azure.Devices.Edge.Test/PriorityQueues.cs

```diff
@@ -187,5 +266,19 @@ private async Task<PriorityQueueTestStatus> PollUntilFinishedAsync(string module
             while (!testStatus.IsFinished);
             return testStatus;
         }
+
+        private TestInfo InitTestInfo(int length, int ttlThreshold, bool defaultTtls = false)
```

Comment:
```suggestion
        private TestInfo InitTestInfo(int ttlSampleLength, int ttlThreshold, bool defaultTtls = false)
```

Reply:
should combine ttlThreshold and defaultTtls into Option<int> ttlThreshold?

Reply:
See answer to your other comment